controls/6_1_system_file_permissions.rb

```diff
@@ -149,8 +149,8 @@
   gshadow_files.each do |f|
     describe file(f) do
       it { should exist }
-      it { should be_readable.by 'owner' }
-      it { should be_writable.by 'owner' }
+      it { should_not be_readable.by 'owner' }
```

Comment:
please add a comment describing our changes and the reason why they are divergent from the original

Reply:
to confirm, remove read access for owner from /etc/shadow and /etc/gshadow and add group and other read access?

Reply:
both of these should be `0000`, so I just added the missing bit to set it group-non-readable in b1c579076ac31b1967c73d8dfd8fb16058bcfe1f